lib/routes/api/metadata/request.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+/**
+ * Dependencies
+ */
+
+const MAX_REDIRECTS = require('../../../../config').max_redirects;
```

Comment:
nit: a bit ugly with all '../../../', can we change it to `__dirname` ? or something similar?
Not having a strong opinion here, so if you find it problematic just leave as it is.